IPython/external/qt_loaders.py

```diff
@@ -110,17 +110,26 @@ def has_binding(api):
                    QT_API_PYQT_DEFAULT: 'PyQt4'}
     module_name = module_name[api]
 
-    import imp
+
+    if sys.version_info.major == 3 and sys.version_info.minor >= 4:
```

Comment:
I think `sys.version_info >= (3, 4)` is neater here. Also future proof for Python 4 ;-)